webdriver/tests/get_computed_role/get.py

```diff
@@ -77,9 +77,9 @@ def test_stale_element_reference(session, stale_element, as_frame):
 
 
 @pytest.mark.parametrize("html,tag,expected", [
-    ("<li role=menuitem>foo", "li", "menuitem"),
+    ("<article>foo</article>", "article", "article"),
     ("<input role=searchbox>", "input", "searchbox"),
-    ("<img role=presentation>", "img", "presentation")])
+    ("<img role=button tabindex=0>", "img", "button")])
```

Comment:
Thanks for the patch! On Firefox side, I had filed https://bugzilla.mozilla.org/show_bug.cgi?id=1828600 to review this test, and I was wondering if the two following test cases would make sense: 

> ("<img role=notavalidrole>", "img", "img"),
Check that an invalid role leads to return the default role for the element

> ("<img role=presentation>", "img", None),
Check that presentation/none effectively remove the element from the accessibility tree

Do you think they would be worth adding? I only checked that they passed with the recent work from @jcsteh , so I don't know if that would work for other vendors?

Reply:
This null vs "" vs "none" thing is being discussed in https://github.com/web-platform-tests/wpt/pull/39618#pullrequestreview-1393840907 also. I probably need to file a separate issue for it.

Reply:
There are also a lot of tests in /wai-aria and /html-aam that will more fully validate this accessibility parts without resulting in spurious failures to this WebDriver getter test.